Makefile

```diff
@@ -1,7 +1,21 @@
+.PHONY: generate validate-sigs install-metrics-binaries lint
+
+export GOLANGCI_LINT_VERSION := v1.62.2
```

Comment:
This is expected to change all the time, therefore, I would recommend not to place it in the makefile.

Reply:
BTW, v2 is already out and is not backward compatible with the configuration. Perhaps it is worth bumping now already.